framework/test/unit/modules/dpos_v2/commands/unlock.spec.ts

```diff
@@ -122,6 +127,56 @@ describe('UnlockCommand', () => {
 		header = testing.createFakeBlockHeader({ height: blockHeight });
 	});
 
+	describe('verify', () => {
+		it('should return an OK verify status', async () => {
+			verifyContext = testing
+				.createTransactionContext({
+					stateStore,
+					transaction,
+					header,
+					chainID,
+					maxHeightCertified: blockHeight,
+				})
+				.createCommandVerifyContext();
+
+			const result = await unlockCommand.verify(verifyContext);
+
+			expect(result.status).toBe(VerifyStatus.OK);
+		});
+
+		// check if transaction params are empty
+		it('should return an error if transaction params are empty', async () => {
```

Comment:
Pretty trivial; I guess message intend to say; 'should return an error if transaction params are not empty'

Reply:
Nice catch @has5aan, you are right 😀